pkg/network/ebpf/c/tracer/telemetry.h

```diff
@@ -28,10 +28,11 @@ enum telemetry_counter {
     tcp_finish_connect_failed_tuple,
     tcp_close_target_failures,
     tcp_done_connection_flush,
-    tcp_close_connection_flush
+    tcp_close_connection_flush,
+    tcp_syn_retransmit,
 };
 
-static __always_inline void increment_telemetry_count(enum telemetry_counter counter_name) {
+static __always_inline void __increment_telemetry_count(enum telemetry_counter counter_name, int times) {
```

Comment:
I wonder if a default value of `1` for `times` would be better here. Do we have the compiler set up to allow that?

Reply:
Wait, didn't read far enough down. What you have is better.